### ISSUE TITLE
Desktop Output Constraints Locked

### ISSUE THREAD

**Issue by MSeal** [nteract/archived-desktop-app#5138]
**Application or Package Used**
State which application or package this affects. For example: nteract desktop, nteract web, nteract on jupyter, etc.

nteract desktop

**Describe the bug**

Can't expand output to fit graphs.

**To Reproduce**

1. Open a python3 notebooks
2. Add 
```
%pip install pandas vega_datasets matplotlib
```
to the first cell and run it
3. Add
```
from vega_datasets import data
import pandas

pandas.set_option('display.html.table_schema', True)
```
and run in the next cell
4. Add
```
data.cars()
```
5. See how data explorer is always within a small windowed box despite window sizing.
6. Try Toggle Expand Output from dropdown, nothing changes.
7. Try with matplot lib and get same resulting boxing
```
p = data.cars().plot(x='Name', y='Miles_per_Gallon', legend=True)
for tick in p.get_xticklabels():
    tick.set_rotation(45)
p.set_dpi(80)
```

**Expected behavior**

The display output to fix within the initial box sizing and for the toggle expand output to work. Ideally it'd be a little smart and expand to a point then go to unexpanded if it were much larger.

**Screenshots**

![data-explorer-box](https://user-images.githubusercontent.com/1863892/83687412-9288a300-a5a0-11ea-8fd6-909b8d137dcf.png)

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04
- Browser Electron
- Version 0.23.3

**Additional context**

**Comment by MSeal** [comment]
Dev build fixes the toggle, though I still think a longer default height before snapping to a fixed height would be a better experience. Classic UI actually does this pretty well imo.

**Comment by captainsafia** [comment]
We're about even with Jupyter classic at the max height that is set for outputs. Jupyter sets the max-height to 336px and we set it to 330px.

I bumped it to match the 336px used by Classic in #https://github.com/nteract/nteract/pull/5142.

**Comment by MSeal** [comment]
True, but in classic it scrolls almost a whole screen height before it hits a threshold and then it snaps back to 336px once it goes past some upper bound of output and isn't expand toggled.

**Comment by captainsafia** [comment]
@MSeal Can you share a gif of what you mean when you have the chance? I'm not seeing the same thing you are describing but I might be misunderstanding something.

**Comment by MSeal** [comment]
![captured](https://user-images.githubusercontent.com/1863892/84095692-cc004a80-a9b4-11ea-9d9a-48be89e8fec0.gif)

**Comment by captainsafia** [comment]
Ah! I see you are referring to the experience when a cell is running.

This should be fairly trivial for us to do. We'll need to add a CSS classname `running` to the outputs that depends on the current execution state. Elements with the `running` class will take up the full height needed.
